profiles/w3c.js

```diff
@@ -33,10 +33,10 @@ const modules = [
   import("../src/core/best-practices.js"),
   import("../src/core/figures.js"),
   import("../src/core/webidl.js"),
-  import("../src/core/data-cite.js"),
   import("../src/core/biblio.js"),
-  import("../src/core/webidl-index.js"),
   import("../src/core/link-to-dfn.js"),
+  import("../src/core/data-cite.js"),
```

Comment:
In near future, we'll have this:
```suggestion
  import("../src/core/link-to-dfn.js"),
  import("../src/core/xref.js"),
  import("../src/core/data-cite.js"),
```